scripts/version-and-build.sh

```diff
@@ -6,11 +6,6 @@ if [ "$CIRCLE_JOB" == "" ]; then
     exit 1
 fi
 
-# Install AWS cli
```

Comment:
These are already installed in our base docker container: https://github.com/dydxprotocol/docker-node/blob/master/Dockerfile#L3